client-v2/src/components/inputs/richtext/MenuView.tsx

```diff
@@ -0,0 +1,100 @@
+import { EditorView } from 'prosemirror-view';
+import { toggleMark } from 'prosemirror-commands';
+import React from 'react';
+import { styled, theme } from 'constants/theme';
+import { basicSchema } from './setup';
+import { icons } from './icons';
+import {
+  linkItemCommand,
+  removeAllMarksFromSelection,
+  unlinkItemCommand
+} from './command-helpers';
+import { EditorState, Transaction } from 'prosemirror-state';
+import { undo, redo } from 'prosemirror-history';
+
+const MenuViewWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const MenuButton = styled.div`
+  width: 30px;
+  height: 25px;
+  padding: 4px;
+  background: transparent;
+  text-align: center;
+  border: 1px solid black;
+  background-color: ${theme.shared.colors.whiteDark};
+  :hover {
+    background-color: ${theme.shared.colors.greyLight};
+    cursor: pointer;
+  }
+  svg {
+    height: 100%;
+    fill: ${theme.shared.colors.blackDark};
+  }
+`;
+
+export const MenuView = ({ edView }: { edView: EditorView }) => {
+  const linkEl = document.createElement('i');
+  linkEl.innerHTML = icons.link;
+
+  const menuItems = [
+    {
+      command: toggleMark(basicSchema.marks.strong),
+      dom: icons.strong,
+      title: 'bold'
+    },
+    {
+      command: toggleMark(basicSchema.marks.em),
+      dom: icons.emphasis,
+      title: 'italic'
+    },
+    {
+      command: linkItemCommand(basicSchema.marks.link)(),
+      dom: icons.link,
+      title: 'add-link'
+    },
+    {
+      command: (state: EditorState, dispatch: (tr: Transaction) => void) =>
+        unlinkItemCommand(basicSchema.marks.link)(state, dispatch),
```

Comment:
If I'm right, this could be simplified to:

```ts
command: unlinkItemCommand(basicSchema.marks.link)
```

Reply:
Ah yeah it can 👍